t/14-isotovideo.t

```diff
@@ -30,8 +30,13 @@ sub isotovideo {
     $args{opts}         //= '';
     $args{exit_code}    //= 1;
     my @cmd = ($^X, "$toplevel_dir/isotovideo", '-d', $args{default_opts}, split(' ', $args{opts}));
-    note("Starting isotovideo with: @cmd");
-    capture [$args{exit_code}], @cmd;
+    note "Starting isotovideo with: @cmd";
+    my $output = qx(@cmd);
```

Comment:
Any idea why `qx` would be more *reliable* than `capture`? I couldn't even repro this with your branch.

Reply:
Looks like a bug in `IPC::Run::Simple` which provides `capture`. However, I'm not really motivated to investigate bugs in `IPC::Run::Simple`, especially if using built-in functions does not even require much more code.

Reply:
And about the reproducibility: I couldn't reproduce it locally as well until today where it actually almost always failed (untils switching to qx). So I'm not sure what "variable" influences whether the bug is triggered or not.